### DISCUSSION TITLE
Using Rfast to sort a 2-D matrix

### DISCUSSION THREAD

**Discussion by DrB-S** [RfastOfficial/Rfast#66] [Q&A]
How can I use Rfast to do a numeric sort of a 2-D matrix in both dimensions, keeping the first column and row as corresponding labels (e.g., col1 label = row2 label, etc.)?

**Comment by ManosPapadakis95** [comment]
I am not user what you want to achieve. Please give an example.

Στις Τρί, 7 Μαρ 2023, 20:55 ο χρήστης Stephen Beckstrom-Sternberg <
***@***.***> έγραψε:
